docs/explanation/priority-inversion-inheritance.rst

```diff
@@ -0,0 +1,67 @@
+Priority inversion and inheritance
+==================================
+
+Among the reasons why PREEMPT_RT is not in mainline yet is that much of the locking within the kernel has to be updated to prevent priority inversion from occurring in a real-time environment.
```

Comment:
Hi @farshidtz , the intro to PREEMPT_RT here is a bit sudden. I know this is from the blog but at that point in the blog there was more context around the real-time kernel itself and PREEMPT_RT.

I suggest prefacing this a bit with a tiny overview of PREEMPT_RT.

Something along the lines of (not the best, please revise as needed):

The PREEMPT_RT patch is a set of Linux kernel patches with the goal of transforming Linux into a real-time system.

Reply:
Good point. I've replaced the intro.